src/core/server/config.ts

```diff
@@ -255,6 +255,14 @@ const config = convict({
     env: "SCRAPE_TIMEOUT",
     arg: "scrapeTimeout",
   },
+  disable_force_ssl: {
+    doc:
+      "Disables forcing SSL in production environments. Should not be used except for testing.",
```

Comment:
is this being used?

Reply:
Yes, right here: https://github.com/coralproject/talk/pull/2736/files#diff-cf797d405858c20b0420663894bf0c4bR133

Mostly designed to be used that when we're testing production branches locally, we might need a way to force off SSL redirects.